consultation_analyser/pipeline/ml_pipeline.py

```diff
@@ -55,15 +55,16 @@ def get_answers_and_topics(topic_model, answers_list: List[Dict[str, Union[UUID,
     # Assign topics to answers
     answers_df = topic_model.get_document_info(free_text_responses)
     answers_df["id"] = answers_id_list
-    answers_df = answers_df[["id", "Name"]]
+    answers_df = answers_df[["id", "Top_n_words", "Topic"]]
     return answers_df
 
 
 def save_themes_to_answers(answers_topics_df: pd.DataFrame) -> None:
     for row in answers_topics_df.itertuples():
         answer = models.Answer.objects.get(id=row.id)
-        theme_label = row.Name
-        answer.save_theme_to_answer(theme_label=theme_label)
+        theme_keywords = row.Top_n_words.split(" - ")
+        theme_number = row.Topic
+        answer.save_theme_to_answer(keywords=theme_keywords, theme_number=theme_number)
```

Comment:
Given the determination of theme number > is_outlier is part of bertopic, should we have that if statement out here, and pass is_outlier to save_theme_to_answer?

Reply:
Yes, the calculation of if it's an outlier has now been moved here.